### PR TITLE
SSCS-9871 refactored common for reissue FE

### PR DESCRIPTION
**Pull request by gmikerlemos**
Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/SSCS-9871

### Change description ###

Child Support - Update Reissue Further Evidence Event

**Does this PR introduce a breaking change?** (check one with "x")

```
[x ] Yes
[ ] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/sscs/service/NotificationServiceTest.java [hunk 1]
```diff
@@ -111,12 +111,12 @@
     private CcdNotificationWrapper ccdNotificationWrapper;
     private SscsCaseDataWrapper sscsCaseDataWrapper;
     private final Subscription subscription = Subscription.builder()
-        .tya(APPEAL_NUMBER)
-        .email(EMAIL)
-        .mobile(MOBILE_NUMBER_1)
-        .subscribeEmail(YES)
-        .subscribeSms(YES).wantSmsNotifications(YES)
-        .build();
+            .tya(APPEAL_NUMBER)
```
Comment (ryandac) [inline src/test/java/uk/gov/hmcts/reform/sscs/service/NotificationServiceTest.java]: There is a lot of formatting changes in this class. Are you using a different intelliJ settings from the rest of the team?

**Review by ryandac — APPROVED** [review]
Why is there so much formatting changes, check your intelliJ project settings

File: src/test/java/uk/gov/hmcts/reform/sscs/service/NotificationServiceTest.java (hunk 1 above)
Comment (gmikerlemos) [reply src/test/java/uk/gov/hmcts/reform/sscs/service/NotificationServiceTest.java]: Same as everyone

Comment (jackmaloney) [reply src/test/java/uk/gov/hmcts/reform/sscs/service/NotificationServiceTest.java]: I don't know, kind of feel these formatting changes should go in their own PRs in future as makes it very difficult to review these PRs when there are so many lines that have 'changed'

**Review by ryandac — APPROVED** [review]

**Pull request merged**